deno.jsonc

```diff
@@ -18,10 +18,10 @@
   "tasks": {
     
     /* Run deno tests with `deno task test`, using default import map */
-    "test": "deno test --lock=deno-lock.json --ignore=test/dist test",
+    "test": "deno test --lock=deno-lock.json --ignore=test/dist,test/asn1.test.js test",
```

Comment:
What's the ignoring necessary for? Isn't the whole point to run the test?

Reply:
I'm not familiar with deno. My assumption is its running a non-node (client side) JS version of the library. The node crypto library isn't available in this case or at least throws an error about when I run my test in deno.

Since webcrypto uses raw/IEEE P1363 signatures, you would not see the error. So I decided to only have the tests run on the node/npm version where the standard library runs into this issue regularly.

However, I'll update the tests to use static fixtures to avoid the dependency on the crypto library (based on your other comment), so this exclusion will not be needed.